views/fda/partials/search-form-fields.html

```diff
@@ -0,0 +1,109 @@
+{% macro addSelectOption(value, currentValues) %}
+  {% if currentValues and value in currentValues %}
+    {% set selected=true %}
+  {% endif %}
+  <option
+    value="{{ value }}"
+    {% if selected %}selected{% endif %}
+    >
+    {{ value }}
+  </option>
+{% endmacro %}
+
+<fieldset class="search-bar">
+  <label>Drug, ingredients, or other keywords</label>
+  <input type="search" title="Search" placeholder="e.g. &quot;heart attack&quot;" name="q" value="{{ query.q }}" autofocus />
```

Comment:
Not sure it matters, but the example value returns no hits and a suggestion to also try searching in the documents text. Might want to change it in something that returns results, like "Aspirin" or smth else.